bin/openplatform

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env php
+<?php
+
+require __DIR__ . '/../vendor/autoload.php';
+
+use DDB\OpenPlatform\OpenPlatform;
+use Symfony\Component\Yaml\Yaml;
+
+$app = new Silly\Application();
+
+$app->command('search [--fields=] token string', function ($fields, $token, $string) {
+    $op = new OpenPlatform($token);
+
+    $req = $op->searchRequest();
+    $req->query = $string;
+    if ($fields) {
+        $req->fields = explode(',', $fields);
+    }
+
+    $res = $req->execute();
+
+    print(Yaml::dump($res->data));
+    print "Hitcount: " . $res->hitCount . "\n";
```

Comment:
Is there a reason why we print `hitCount` explicitly? I thought it would be a part of `data`.

Reply:
Oh no it's not. It's next to `statusCode`, `errors` and `more`. As far as I can se `data` contains the "thing(s)" while metadata is  at the root.

Reply:
Then how can I tell from the codebase (in the current form) that these properties exist? Should they be added as `@property`s in the response class DocBlock?

Reply:
Oh, right, i forgot the `@property` blocks...